routing/src/frrmi.rs

```diff
@@ -332,7 +372,8 @@ impl Frrmi {
             self.stats.last_fail_genid = Some(response.genid);
             self.stats.apply_failures += 1;
             let out = response.get_response_data();
-            error!("Failed to apply FRR configuration: {out}");
+            error!("Failed to apply FRR configuration for gen {respgen}: {out}");
+            self.config_retry(request);
```

Comment:
The commit says that frrmi does not try to reapply configuration but this looks like it does?  Am I reading this wrong?

Also, is there no case where there can be an intermittent failure of frr-reload to apply the config but a retry would succeed?